e2e/cypress/integration/enterprise/ldap/ldap_login_spec.js

```diff
@@ -115,7 +115,7 @@ context('ldap', () => {
             cy.apiAdminLogin().then(() => {
                 cy.apiUpdateConfig(ldapSetting).then(() => {
                     cy.doLDAPLogin(testSettings).then(() => {
-                        cy.get('#createPublicChannel').should('be.visible');
+                        cy.findAllByLabelText('town square public channel').should('be.visible');
```

Comment:
@saturninoabril I'm getting this failure when I run the test locally

![Screen Shot 2020-08-12 at 10 27 08 AM](https://user-images.githubusercontent.com/487991/90047346-9f9fcd80-dc86-11ea-9856-20f402d58eb9.png)

Reply:
Thanks for checking. Pass on me but not sure why it's failing on you. I added a URL check and timeout. Please try again and let me know how it works for you.

Reply:
That fixed it